packages/output/src/browser/output-channel-manager-client.ts

```diff
@@ -0,0 +1,54 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { Emitter } from '@theia/core';
+import { injectable, inject } from 'inversify';
+import { OutputPreferences } from '../common/output-preferences';
+import { OutputChannel, OutputChannelManager, OutputOptions } from '../common/output-channel';
+
+@injectable()
+export class OutputChannelManagerClient implements OutputChannelManager {
+    protected readonly channels = new Map<string, OutputChannel>();
+
+    private readonly channelDeleteEmitter = new Emitter<{ channelName: string }>();
+    private readonly channelAddedEmitter = new Emitter<OutputChannel>();
+    readonly onChannelDelete = this.channelDeleteEmitter.event;
```

Comment:
please follow event naming schema from project naming conventions

Reply:
Those emitters were moved from OutputChannelManager (as OutputChannelManager is now an interface and this class is now the implementation).  I left the names alone to reduce incompatibility, and kept the pattern for the emitters in OutputChannelReaders.  However we are already breaking implementations with their own output channel viewers, so I have changed the names as requested (onDidDeleteChannel, onDidAddChannel).